### PR TITLE
Upgrade lodash to version 3.10.1

### PR DESCRIPTION
**Pull request by rolandpoulter**
Depends on https://github.com/RackHD/on-core/pull/61

Tests should fail until `on-core` PR is merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +24,7 @@
     "express": "^4.10.7",
     "express-http-proxy": "^0.6.0",
     "glob": "^4.3.2",
-    "lodash": "^2.4.1",
-    "lru-cache": "^2.5.0",
```
Comment (rolandpoulter) [inline package.json]: Removed `lru-cache` because it was not being used and is already included by `on-core`

File: index.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ function run() {
         });
 
     process.on('SIGINT', function() {
-        http.stop()
```
Comment (rolandpoulter) [inline index.js]: This was causing an uncaught exception when the process was interrupted.

Comment (benbp) [inline index.js]: Thanks, I was noticing this too but forgot to add it to my last PR.

**Comment by benbp** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**